Formula/gts.rb

```diff
@@ -14,7 +15,13 @@ class Gts < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:  "8574a923a6854116b836f04ffdd0b556d71c3e63da1127a90c2e6ac573900c88"
   end
 
-  depends_on "pkg-config" => :build
+  # We regenerate configure to avoid the `-flat_namespace` flag.
```

Comment:
I think this was supposed to be on a different line?

Reply:
No, I did mean to put this here. I was trying to explain the Autotools dependencies.

Reply:
Ah, wait, I see what you mean. `brew style --fix` moved my comment 😅